resources/web/style/toc.pcss

```diff
@@ -1,9 +1,13 @@
 #guide {
+  .book-title {
+    font-size: 120%;
+    font-weight: 600;
+    margin: 20px 0px 20px 20px;
+  }
```

Comment:
```suggestion
```

Reply:
I think we should instead target the id `book_title`

<img width="304" alt="Screen Shot 2022-07-20 at 1 25 36 PM" src="https://user-images.githubusercontent.com/5618806/180075178-53df7cf9-e116-49d3-a46d-711fa117e49c.png">